src/dataset/mod.rs

```diff
@@ -22,6 +22,8 @@ pub struct Dataset {
     pub source_url: String,
     pub resources: Vec<Resource>,
     pub issued: Option<Date>,
+    pub comment: Option<String>,
+    pub last_checked: Option<Date>,
```

Comment:
Could we group the fields as much as this is possible, i.e. `issued` and `last_checked` are both "time fields" in a broad sense where `comment` seems to belong close to `title` and `description`?

Reply:
What I just noticed now: Do we really need to whether a comment is present or not in contrast to an empty comment? (The reason I am asking is consistency, the `description` field is not an `Option` even though it is optional, meaning it is just left empty if not present.)

Reply:
This indeed should be handled consistently. I would prefer to use Option whenever possible, so we can easier keep track of if something was not filled by us or the source.

Reply:
Ok, then `description: Option<String>` is the way to go I guess.

Reply:
I have implemented this. There might be a problem with they way I handled the doris harvester, though. I works but is ugly.

Reply:
I have made comments at the relevant places. I still think it would be nicer to put comment next to title and description.

A related question is whether we want to index and search comment by default like we do for title and description? Or at least index it and make it available for field-specific search queries?